### DISCUSSION TITLE
IED's and Minefields

### DISCUSSION THREAD

**Discussion by KDaum50** [Vdauphin/HeartsAndMinds#1393] [Question and answer]
I'm wanting to add a certain type of IED's to make it more realistic so there's not just random stuff everywhere. Here is the IED's I'm wanting to replace the existing ones in code: "IEDLandBig_F","ACE_IEDLandBig_Range","IEDUrbanBig_F", "ACE_IEDUrbanBig_Range", "IEDLandSmall_F", "ACE_IEDLandSmall_Range", "IEDUrbanSmall_F", "ACE_IEDUrbanSmall_Range" where can I put these? and is it possible to get rid of fake IED's?

**Comment by Vdauphin** [comment]
Hello,

> where can I put these?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L398

> and is it possible to get rid of fake IED's?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/ied/initArea.sqf#L61
Replace `false` to `true`

Cheers

**Reply by KDaum50** [reply]
Sweet thank you got rid of the fakes now how do i get rid of the random junks that spawns next to the IED? and get rid of the Graffiti that spawns on the road?
![20220812173432_1](https://user-images.githubusercontent.com/110949312/184453424-faa55943-19d8-4717-876a-1a896b235eab.jpg)

**Reply by Vdauphin** [reply]
I don't understand, this is real life stuff

**Reply by KDaum50** [reply]
The issue is we want it to be extremely difficult to find the IED and with the junk spawning next to the IED is like the bad guys waving a flag saying here I am lol

**Reply by Vdauphin** [reply]
Not at all, junk are element from the map and could be fake so it is not obvious 
It is like in real life where insurgent put IED in random objects around road and field
https://www.youtube.com/watch?v=cjva7OXdwJE

**Reply by KDaum50** [reply]
I'm not disagreeing, I'm just saying in Arma the way it Spawns in it just points out the IED which takes out the fun in looking for an IED so if there a way to disable the junk spawning around the IED or at least be able to change what junk spawns in?

**Reply by Vdauphin** [reply]
The IED is hidden inside the floor so I don't see how you can say a junk is an IED

> at least be able to change what junk spawns in?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L342-L356
